gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -144,6 +144,7 @@
             <message key="import.illegal" value="math is deprecated, use math3 instead."/>
         </module>
         <module name="IllegalImport">
+            <property name="id" value="BanLoggingImplementations"/>
```

Comment:
I added this one because a couple of our internal logging libraries needed to suppress this occasionally.